### ISSUE TITLE
Save file rotation

### ISSUE THREAD

**Issue by ain** [tmux-plugins/tmux-resurrect#136]
It'd be nice to have the `*.txt` files rotating, e.g. with only 25 last sessions remaining. To not to overwhelm the storage folder.

**Comment by jarosser06** [comment]
:+1:

**Comment by hprem** [comment]
![](https://assets-cdn.github.com/images/icons/emoji/unicode/1f44d.png)

**Comment by janzenz** [comment]
Agree. I think this is going to be a neat feature. I manually delete the sessions by hand atm.

**Comment by x4121** [comment]
In the meantime you can fix this with an crontab entry of:
`0 * * * * cd $HOME/.tmux/resurrect && ls -A1t | tail -n +12 | xargs -r rm`.
Replace **11** in `tail -n +12` with `$maxSessions + 2`.

`crontab -e` and append at the end of file.
This will run every hour and delete old session files in `$HOME/.tmux/resurrect`.

**Comment by janzenz** [comment]
@x4121 boom! Thanks heaps! I got 300+ .txt files before you posted this!

For OSX this worked for me:
`cd ~/.tmux/resurrect && ls -A1t ~/.tmux/resurrect/ | tail -n +12 | xargs rm`
PS. `cd` to make sure you are on the right directory.

**Comment by x4121** [comment]
Even OSX should have crontab (typing `crontab -e` in a terminal should open your user's crontab in vi or some other editor), at least some colleagues confirmed this, don't run OSX myself.

**Comment by janzenz** [comment]
@x4121 Thanks but since there is no ETD for the fix I was thinking using `launchd` will address this issue "for good" just in case I forget to migrate from `crontab` to `launchd` in the future.

**Comment by janzenz** [comment]
For OSX Users I've already done the [dirty work you](https://gist.github.com/janzenz/146f1ae3e3ec9bf9fe99146f6079deee)
